app/src/main/java/org/oppia/app/player/state/StateFragmentPresenter.kt

```diff
@@ -81,4 +91,25 @@ class StateFragmentPresenter @Inject constructor(
   fun setAudioFragmentVisible(isVisible: Boolean) {
     getStateViewModel().setAudioFragmentVisible(isVisible)
   }
+
+  private fun subscribeToCurrentState() {
+    ephemeralStateLiveData.observe(fragment, Observer<EphemeralState> { result ->
+      logger.d("StateFragment", "getCurrentState: " + result.state.name)
```

Comment:
Prefer: ``"getCurrentState: ${result.state.name}"`` since it's a bit more readable, and idiomatic to Kotlin.

Reply:
Changing this as per kotlin code.